kevinene91/Store-Manager-API
# #161209876  add products feature

### What does this PR do? 
It has the products endpoints 
` POST /api/v1/products`
`GET /api/v1/products`
`GET /api/v1/products/1`
`PUT /api/v1/products/1`
`DEL /api/v1/products/1`
 
### Description of Task completed?
Add products for sale
### How to Manually test this?
 
- Clone the repository `git clone https://github.com/kevinene91/Store-Manager-API.git`
- Change into the app directory `cd Store-Manager-API`
- get this branch code `git fetch origin ft-products-161209876`
- Navigate the folder structure
- create a virtual environment `python3 -m pyvenv venv`
- pip install -r requirements.txt
- export SECRET_KEY ="random string here"
-  run `python run.py`
- Test on Postman 
- Navigate folder structure 
app/tests/v1 run `pytest test_products.py` should pass 
### Relevant PVT story? 

[161209876](https://www.pivotaltracker.com/n/projects/2203367/stories/161209876)
